.github/dependabot.yml

```diff
@@ -36,10 +36,6 @@ updates:
         patterns:
           - "@xterm/*"
     ignore:
-      # https://github.com/cockpit-project/cockpit/issues/21151
-      - dependency-name: "sass"
-        versions: ["1.x", "2.x"]
-
```

Comment:
Doing a quick search in our Cockpit org I see a lot  of `@import` left in [cockpit](https://github.com/cockpit-project/cockpit/blob/86e3bbbc09f4f303965cb2678fa0efa336e57e5d/pkg/lib/console.css#L1), machines, podman and files. Shall we fix that first before removing the sass dependency.